Source/Microsoft.Teams.Apps.FAQPlusPlus.Common/Models/ConfigurationEntity.cs

```diff
@@ -0,0 +1,21 @@
+﻿// <copyright file="ConfigurationEntity.cs" company="Microsoft">
+// Copyright (c) Microsoft. All rights reserved.
+// </copyright>
+
+namespace Microsoft.Teams.Apps.FAQPlusPlus.Common.Models
+{
+    using Microsoft.WindowsAzure.Storage.Table;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Represents configuration entity used for storage and retrieval.
+    /// </summary>
+    public class ConfigurationEntity : TableEntity
```

Comment:
Make this internal so it's not used outside Common project [Not blocking]

Reply:
done